mentat/diff_context.py

```diff
@@ -109,15 +111,22 @@ def files(self) -> list[Path]:
         return self._files_cache
 
     @classmethod
-    def create(
+    async def create(
         cls,
         diff: Optional[str] = None,
         pr_diff: Optional[str] = None,
     ):
+        stream = SESSION_STREAM.get()
         git_root = GIT_ROOT.get()
 
         if diff and pr_diff:
-            raise UserError("Cannot specify more than one type of diff.")
+            await stream.send(
+                "Cannot specify more than one type of diff. Disabling diff and"
+                " pr-diff.",
+                color="light_yellow",
+            )
+            diff = None
+            pr_diff = None
```

Comment:
Is it ok to not raise a `UserError` here anymore? Assuming the user mistakenly puts in both `diff` and `pr_diff` (and just meant to put in one or the other) idk if it makes sense to continue running and call `merge-base` later with no diffs.

Reply:
Well for one, we can't actually send a message in the startup of Session yet (where this is), since Session is created before the prompt toolkit stuff is (because the prompttoolkit stuff uses the config contextvar); I added a TODO, which I plan on doing soon, to move the config to the client, which means we'll set up listening for the stream before we start the session, which means we'll actually be able to send a message and quit here (and other places). I actually prefer this to just outright quitting though, having no diff and no pr-diff is normal (it's what happens when you just call mentat without either argument) and it gives mentat the current git diff from the HEAD. If you think we should still quit though, I'm totally ok with that once we solve the config problem

Reply:
makes sense to me!